config-scripts/cups-compiler.m4

```diff
@@ -114,8 +114,8 @@ AS_IF([test -n "$GCC"], [
 
     # Default optimization options...
     AS_IF([test -z "$OPTIM"], [
-	# Default to optimize-for-size and debug
-	OPTIM="-Os -g"
+	# Default to optimize-for-size
```

Comment:
We still want debug symbols. Most Linux distributions will generate a debug package to help developers and then strip.